src/components/modal/__mocks__/tabbable.ts

```diff
@@ -0,0 +1,11 @@
+const lib = jest.requireActual('tabbable');
+
+const tabbable = {
+  ...lib,
+  tabbable: (node, options) => lib.tabbable(node, { ...options, displayCheck: 'none' }),
+  focusable: (node, options) => lib.focusable(node, { ...options, displayCheck: 'none' }),
+  isFocusable: (node, options) => lib.isFocusable(node, { ...options, displayCheck: 'none' }),
+  isTabbable: (node, options) => lib.isTabbable(node, { ...options, displayCheck: 'none' }),
+};
+
+module.exports = tabbable;
```

Comment:
with latest react-topology package, the focus-trap module is updated and it seems to have a known issue in testing environment (jsdom) [Refer](https://github.com/focus-trap/tabbable#testing-in-jsdom)

Mocking this module to avoid the following error in our Modal related tests :
```
Your focus-trap must have at least one container with at least one tabbable node in it at all times
```
cc: @christianvogt @rohitkrai03